src/utxobased/keymanager/coinmapper.ts

```diff
@@ -0,0 +1,13 @@
+import { Coin } from './coin'
+import { Bitcoin } from './coins/bitcoin'
+import { Litecoin } from './coins/litecoin'
+
+const coinClasses: Coin[] = [new Litecoin(), new Bitcoin()]
+
+export function getCoinFromString(coinName: string): Coin {
+  const selectedCoin = coinClasses.filter(coin => coin.name === coinName)
```

Comment:
the find function would be more readable than the filter function. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find